sssd_test_framework/roles/ipa.py

```diff
@@ -1597,3 +1635,103 @@ def __get_info(self, info: str | NFSExport | IPAAutomountMap | None) -> str | No
             return info.name
 
         return info
+
+
+class IPAPasswordPolicy(IPAObject, GenericPasswordPolicy):
+    """
+    Password policy management.
+    """
+
+    def __init__(self, role: IPA, name: str = "ipausers"):
+        """
+        :param role: IPA host object.
+        :type role: IPAHost
+        :param name: Name of target object, defaults to 'ipausers'.
+        :type name: str
+        """
+        super().__init__(role, name, command_group="pwpolicy")
+
+    def complexity(self, enable: bool) -> IPAPasswordPolicy:
+        """
+        Enable or disable password complexity.
+
+        :param enable: Enable or disable password complexity.
+        :type enable: bool
+        :return: IPAPasswordPolicy object.
+        :rtype: IPAPasswordPolicy
+        """
+        if enable and self.get() is None:
+            attrs: CLIBuilderArgs = {
+                "dictcheck": (self.cli.option.VALUE, "True"),
+                "usercheck": (self.cli.option.VALUE, "True"),
+                "minlength": (self.cli.option.VALUE, 8),
+                "minclasses": (self.cli.option.VALUE, 5),
+                "priority": (self.cli.option.VALUE, 1),
+            }
+            self._add(attrs)
+        else:
+            _attrs: CLIBuilderArgs = {
```

Comment:
Is there a need for `_attrs` here vs `attrs` in the conditional section above the else?

Reply:
This was a mypy complaint. 

```
Found 1 error in 1 file (checked 46 source files)
sssd_test_framework/roles/ipa.py:1673: error: Name "attrs" already defined on line 1664  [no-redef]
lint-upstream: exit 1 (0.82 seconds) /home/dlavu/git/sssd-test-framework> mypy --install-types --non-interactive sssd_test_framework tests pid=692404
lint-upstream: FAIL ✖ in 3.71 seconds
```

Reply:
Would defining `attrs` before the conditional as an empty dict solve this?   It seems odd to have to define a new variable here.

Reply:
Indeed, that would help.